tests/baselines/reference/capturedParametersInInitializers2.js

```diff
@@ -36,14 +36,15 @@ function foo(y, x, z) {
     y.c;
 }
 function foo2(y, x) {
-    var _a, _b;
-    if (y === void 0) { y = (_b = /** @class */ (function () {
+    var _a;

```

Comment:
Is this because we treat one of these as a block-scoped variable, but we're downleveling to ES5 so it gets converted to a `var`?

Reply:
Yes, we first split up into `let` and `var` declarations then the down-level for `es5` will convert the let with a var and there is  no logic to merge those back together.